delivery_carrier_option/models/stock_picking.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2012-2015 Akretion <http://www.akretion.com>.
+# Copyright 2013-2016 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    option_ids = fields.Many2many(
+        comodel_name="delivery.carrier.option",
+        string="Options",
+        compute="_compute_option_ids",
+        store=True,
+        readonly=False,
+    )
```

Comment:
I wonder if reusing an overriden "paper.format" should avoid to create this model. And if a new configuration is needed, a new format should be created. This could decoupled the postlogictic's logic to a wider reusable concept (for other implementations like QuickPAC, etc)

Reply:
It was an idea to separated concerns.
- Option for label printing reusing standard printing model (A4, PDF, 600dpi, etc)
- Operational delivery services (Priority, With signature, etc) -> the "real" delivery options
But i presume this could maybe increase functional usage. Nevermind